OrbitBase/Path.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/Path.h"
+
+#include <filesystem>
+#include <string>
+
+#include "absl/strings/str_replace.h"
+
+namespace OrbitBase {
+
+std::string GetExecutableDir() { return GetDirectory(GetExecutablePath()); }
```

Comment:
This feels slightly out of place here as the other functions are "generic" path-related functions, while this returns a very specific file/directory. Does this even belong in `OrbitBase`?

Reply:
This is widely used by tests for pretty much all components so I think it belongs here.